crates/sncast/src/starknet_commands/script/run.rs

```diff
@@ -454,9 +454,7 @@ fn inject_lib_artifact(
         .target_dir
         .clone()
         .unwrap_or_else(|| metadata.workspace.root.join("target"));
-    let sierra_path = &target_dir
-        .join(&metadata.current_profile)
-        .join(sierra_filename);
+    let sierra_path = &target_dir.join("dev").join(sierra_filename);
```

Comment:
Let's also add `TODO #2042` here, so we won't forget to update it.

Reply:
done in newest commit